src/proto/FunctionRpc.proto

```diff
@@ -201,6 +205,16 @@ message FunctionEnvironmentReloadResponse {
   StatusResult result = 3;
 }
 
+// Tell the out-of-proc worker to close any shared memory maps it allocated for given invocation
+message CloseSharedMemoryResourcesRequest {
+  string invocation_id = 1;
```

Comment:
Should we have a list of the memory maps to close? So potentialy we could ask to close N maps.

Reply:
The list is already maintained at the worker. We just need to specify which invocation's maps need to be deleted and the corresponding maps are obtained from the worker's list and the worker will close them all. (refer to the PR in the python worker repo.)

Reply:
We are sending a list of maps to close now. The result is still a binary if the closing was successful or not. Since we are not doing any retries etc, I think it's fine to keep it this way. The worker will log any failures (which the host does get) so they will be emitted to Kusto.